housekeeper/store/api/handlers/create.py

```diff
@@ -107,15 +115,24 @@ def add_file(
         self,
         file_path: Path,
         bundle: Bundle,
+        root: Path,
+        exclude: bool = False,
         to_archive: bool = False,
         tags: List[str] = None,
     ) -> File:
-        """Build a new file object and add it to the latest version of an existing bundle"""
+        """Build a new file object and add it to the latest version of an existing bundle."""
         version_obj = bundle.versions[0]
         tags = tags or []
         tag_objs = [tag_obj for tag_name, tag_obj in self._build_tags(tags).items()]
+        file_path_to_use: str = str(file_path.absolute())
+        if not exclude:
+            file_path_to_use = str(
+                _link_and_convert_to_relative_path(
+                    file_path=file_path, root_path=root, version=version_obj
+                )
+            )
         new_file = self.new_file(
-            path=str(file_path.absolute()),
+            path=file_path_to_use,
```

Comment:
The add_file function is used quite a bit in cg and I'm not super comfortable changing it. Can we create an `include_file` function that does the linking part and then invokes this function?

Reply:
Yes, and that is the one we need ot streamline together with how the implentation is in housekeeper

Reply:
So potential fix needed in cg too?